---
layout: post
title: "The applications behind Cappable-seq"
date: "2016-03-15 12:02:16 -0400"
categories: jekyll update
published: true
---





[Cappable-seq][Cappable-seq] is a new technology to capture in a specific manner 5' triphostphate RNA. Triphosphate RNA are universally found in nature but only a specific subset of RNA are triphosphorylated; those are primary transcripts prior to any post-transcriptional modifications on the 5'end. 

We have used this technology to precisely identify transcription start sites at base resolution in culture [bacteria][Cappable-seq] and outperform all existing technologies for TSS determination in prokaryotes. To obtain the TSS, we shear the total RNA prior to capture and only retain the 5' ends that are triphosphorylated. We then directionally sequence the first 75 bp providing a base resolution map of TSS genomewide.

Apply to a microbiome, Cappable-seq perform as well as in isolated culture of bacteria and reveals TSS across bacteria kindom. For TSS determination, the quality of the RNA matters little and can be heavily degraded. We think that is the reason Cappable-seq works so well on a microbiome. The majority of the reads are likely to be the first 75 base 5' of meta-transcripts. 

Because ribosomal transcripts are typically processed, Cappable-seq has the added bonus to naturally remove ribosomal transcripts regardeless of sequence homology. Ribosomal RNA only account for ~5% of the reads after Cappable-seq instead of the 80-90% typically achieve if no ribosomal removal technologies are used. This is true for isolated culture or complex microbiome. 



What makes this technology so appealing is its versatility of applications. or example ribosomal RNA is universally removed. 







[Cappable-seq]: http://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2539-z
